irohad/main/irohad.cpp

```diff
@@ -313,6 +313,12 @@ int main(int argc, char *argv[]) {
       return EXIT_FAILURE;
     }
 
+    if (config.max_past_created_time)
+    {
+        shared_model::validation::FieldValidator::kMaxDelay =
+                std::chrono::milliseconds(std::chrono::hours(config.max_past_created_time.value()) / std::chrono::milliseconds(1));
```

Comment:
@iceseer But it is not global variable - it is static variable from [here](https://github.com/hyperledger/iroha/blob/57400a9ef7a65a58c6b99a9a7ee0dea9c36c16b7/shared_model/validators/field_validator.hpp#L207). Which global variable should I remove?

Reply:
Statics are not considered good in practice. In the code-base it should be easy to track where a value came from.

Reply:
Thanks @appetrosyan for the comment. @iceseer explained me how to solve the issue without statics. It is not that easy, that is why I haven't repair this already. It can be solved, but it requires debugging of 30+ objects creation to find which one is in proper place:). I'll repair its probably after internship in September:)

Reply:
@appetrosyan I've managed to find the connection between config file and validators without statics.